File: app/src/main/java/com/github/kotvertolet/youtubeaudioplayer/data/models/youtube/videos/list/RegionRestriction.java
package com.github.kotvertolet.youtubeaudioplayer.data.models.youtube.videos.list;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class RegionRestriction {

    @SerializedName("allowed")
    private List<String> allowed;

    public List<String> getAllowed() {
        return allowed;
    }

    public void setAllowed(List<String> allowed) {
        this.allowed = allowed;
    }

    @Override
    public String toString() {
        return
                "RegionRestriction{" +
                        "allowed = '" + allowed + '\'' +
                        "}";
    }
}

File: app/src/main/java/com/github/kotvertolet/youtubeaudioplayer/data/models/youtube/videos/list/RegionRestriction.kt
package com.github.kotvertolet.youtubeaudioplayer.data.models.youtube.videos.list

import com.google.gson.annotations.SerializedName

class RegionRestriction {
    @SerializedName("allowed")
    var allowed: List<String>? = null

    override fun toString(): String {
        return "RegionRestriction{" +
                "allowed = '" + allowed + '\'' +
                "}"
    }
}